tests/app/test_surveys.py

```diff
@@ -89,17 +90,25 @@ def test_surveys_todo_fail(self, mock_request):
     def test_access_survey(self, mock_request):
```

Comment:
Is it worth calling this `test_access_survey_seft` to differentiate between the access eq tests. Just to make it clearer?